### DISCUSSION TITLE
Math is not rendering

### DISCUSSION THREAD

**Discussion by sandeep026** [jgraph/drawio#4394] [Q&A] [answered]
Sometimes, math is not rendered by the software (15.8.7).
![image](https://github.com/jgraph/drawio/assets/16457676/8903b8a3-aff7-4659-9b0b-1fe98ccd04ab)

**Comment by davidjgraph** [comment]
15.8.7 is a 2021 release. You are welcome to report issues, but they need to be on the latest version and in the form of a proper bug report.

**Reply by sandeep026** [reply, marked as answer]
This is due to formatting done by draw.io when math is copy pasted. Using "clear formatting" from text tab removes this issue.
